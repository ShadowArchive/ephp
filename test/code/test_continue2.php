<?php

$a = [1,2,3,4,5,6,7,8,9,10];
$b = ["one", "two"];

foreach ($b as $k) {
    foreach ($a as $i)
        foreach ($a as $j) {
            print "($k) $i x $j = ";
            if ($i * $j > 8) {
                print "greater than 8!\n";
                continue 2;
            }
            print ($i * $j) . "\n";
        }
}

print "OK!\n";
