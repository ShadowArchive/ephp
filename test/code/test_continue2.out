(one) 1 x 1 = 1
(one) 1 x 2 = 2
(one) 1 x 3 = 3
(one) 1 x 4 = 4
(one) 1 x 5 = 5
(one) 1 x 6 = 6
(one) 1 x 7 = 7
(one) 1 x 8 = 8
(one) 1 x 9 = greater than 8!
(one) 2 x 1 = 2
(one) 2 x 2 = 4
(one) 2 x 3 = 6
(one) 2 x 4 = 8
(one) 2 x 5 = greater than 8!
(one) 3 x 1 = 3
(one) 3 x 2 = 6
(one) 3 x 3 = greater than 8!
(one) 4 x 1 = 4
(one) 4 x 2 = 8
(one) 4 x 3 = greater than 8!
(one) 5 x 1 = 5
(one) 5 x 2 = greater than 8!
(one) 6 x 1 = 6
(one) 6 x 2 = greater than 8!
(one) 7 x 1 = 7
(one) 7 x 2 = greater than 8!
(one) 8 x 1 = 8
(one) 8 x 2 = greater than 8!
(one) 9 x 1 = greater than 8!
(one) 10 x 1 = greater than 8!
(two) 1 x 1 = 1
(two) 1 x 2 = 2
(two) 1 x 3 = 3
(two) 1 x 4 = 4
(two) 1 x 5 = 5
(two) 1 x 6 = 6
(two) 1 x 7 = 7
(two) 1 x 8 = 8
(two) 1 x 9 = greater than 8!
(two) 2 x 1 = 2
(two) 2 x 2 = 4
(two) 2 x 3 = 6
(two) 2 x 4 = 8
(two) 2 x 5 = greater than 8!
(two) 3 x 1 = 3
(two) 3 x 2 = 6
(two) 3 x 3 = greater than 8!
(two) 4 x 1 = 4
(two) 4 x 2 = 8
(two) 4 x 3 = greater than 8!
(two) 5 x 1 = 5
(two) 5 x 2 = greater than 8!
(two) 6 x 1 = 6
(two) 6 x 2 = greater than 8!
(two) 7 x 1 = 7
(two) 7 x 2 = greater than 8!
(two) 8 x 1 = 8
(two) 8 x 2 = greater than 8!
(two) 9 x 1 = greater than 8!
(two) 10 x 1 = greater than 8!
OK!
